staging/cert-manager-setup/templates/v1upgrade.yaml

```diff
@@ -0,0 +1,118 @@
+{{- if .Values.installCertManager }}
```

Comment:
Would it work to add the conditional to only run backup/restore `{{- if (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1alpha1") -}}` here instead of doing it in the script?

Reply:
(referencing this part of the script https://github.com/jr0d/cert-manager-upgrade/blob/master/pkg/app/backup.go#L29)

Reply:
yes it would.. and also not create jobs needlessly.